spyderlib/app/spyder.py

```diff
@@ -90,6 +90,12 @@
 # Avoid a bug in Qt: https://bugreports.qt.io/browse/QTBUG-46720
 from qtpy import QtWebEngineWidgets  # analysis:ignore
 
+#==============================================================================
+# Proper high DPI scaling is available in Qt >= 5.6.0. This attibute must
+# be set before creating the application
+#==============================================================================
+if QT_VERSION >= 0x050600:
+    QCoreApplication.setAttribute(Qt.AA_EnableHighDpiScaling, True)

```

Comment:
@patstew, we prefer to avoid a comparison with an hex number, so please write this code like

```python
if hasattr(Qt, 'AA_EnableHighDpiScaling'):
    QCoreApplication.setAttribute(Qt.AA_EnableHighDpiScaling, True)
```